src/main/resources/addon.yml

```
@@ -0,0 +1,6 @@
+name: IslandFly
+version: ${project.version}
+main: bentobox.addon.islandfly.IslandFlyAddon
```

Ah! You're not using the "package format" for official addons.
It should rather be:
```suggestion
main: world.bentobox.islandfly.IslandFlyAddon
```

---

I changed the package name to world.bentobox